### PR TITLE
A11y: provide notifications to sr users via aria-live regions

### PR DESCRIPTION
**Pull request by a2hsh**
## Summary:

1. Created a new object, called A11y in the store.
2. This object has a notifications object, which has three values:

   - enabled: boolean.
   - assertiveNotification: string
   - politeNotification: string
3. In the store, I created a getter and setter for the notification:

   - the getter receives the notification type as parameter, and returns the notification.
   - the setter receives two parameters: the type, and notification, as strings.

4. I created a component and used it at the botom of the page. This component uses the getter from the store to display the latest notification. It has two divs that have the class .screenreader-only, an assertive notification div, and a polite notification div.
5. Whenever a component wants to display a notification, it calls the store.setAccessibilityNotification(type, notification) method, and the notification would be read automatically ❤️

> ## Note:
> Assertive notifications are highly important notifications that interrupts the screen reader. They should be used for critical > messages only.
> Polite notifications however, are placed in the screen reader queue. So they get read when the screen stops reading the >>> current statement (if any). They should be used for non-critical status updates.

This PR is ready for review. Thanks!

### CODE REVIEW

File: src/components/A11yNotifications.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        },
+        politeNotification() {
+            return store.getAccessibilityNotification("polite");
+        },
+    },
+};
+</script>
+<style scoped>
+.screenreader-only {
+    position: absolute;
```
Comment (FrostyCoolSlug) [inline src/components/A11yNotifications.vue]: You shouldn't need to define this, there's a .screenreader-only class at the top level in a global scope, would be best to use that.

File: src/components/GoXLR.vue [hunk 2]
```diff
@@ -1,127 +1,129 @@
...
+                        <Routing />
+                    </ContentContainer>
+                </Tab>
+                <Tab name="System">
+                    <ContentContainer>
+                        <SystemComponent />
+                    </ContentContainer>
+                </Tab>
+            </Tabs>
+            <A11yNotifications />
```
Comment (FrostyCoolSlug) [inline src/components/GoXLR.vue]: Would it be possible to move this down 2 lines so it's available on the device selection page as well (basically push it into the global scope)

**Review by FrostyCoolSlug — CHANGES_REQUESTED** [review]
Couple of points, looks pretty good though!

**Comment by a2hsh** [comment]
Thanks @FrostyCoolSlug 
Fixed and awaiting your feedback

**Review by FrostyCoolSlug — APPROVED** [review]

**Pull request merged**